src/pages/ComponentsPage/ComponentsPage.sass

```diff
@@ -1,3 +1,12 @@
 @import '../../styles/core'
 
 .ComponentsPage
+    .control-bar
```

Comment:
That code was leftover from the AEM component, and doesn't end up affecting the CompnentsPage anymore. Removed.